nix/checks/routers.nix

```diff
@@ -53,29 +75,64 @@
           4 # conference <-> expo
         ];
         scale-network = {
+          base.enable = true;
           router.expo.enable = true;
+          services.frr.enable = true;
+          services.frr.router-id = "10.1.2.3";
+          services.frr.passive-interfaces = [
+            "eth0"
+          ];
+          services.frr.broadcast-interface = [
+            "eth1"
+            "eth2"
+          ];
         };
+        networking.firewall.enable = false;
       };
   };
 
   testScript =
-    { nodes, ... }:
+    { ... }:
     ''
       start_all()
       border.wait_for_unit("multi-user.target")
       conference.wait_for_unit("multi-user.target")
       expo.wait_for_unit("multi-user.target")
 
-      # border can hit both routers
+      # layer 2
+      print("LAYER 2")
+
+      # border can ping both routers
       border.succeed("ping -c 5 10.1.1.2")
       border.succeed("ping -c 5 10.1.2.3")
 
-      # conference should hit neighbors
+      # conference can ping both routers
       conference.succeed("ping -c 5 10.1.1.1")
       conference.succeed("ping -c 5 10.1.3.3")
 
-      # expo should hit neighbors
+      # expo can ping both routers
       expo.succeed("ping -c 5 10.1.2.1")
       expo.succeed("ping -c 5 10.1.3.2")
+
+      # layer 3
+      print("LAYER 3")
```

Comment:
@djacu and I added some checks here for frr and the routes that are learned by each router via ospf